getting_started/dedicated_administrators.adoc

```diff
@@ -235,6 +248,9 @@ resources and objects are consumed by users across the cluster.
 Defaults can be set for quotas and limit ranges for new projects at creation. To
 request such a change, open a support case on the
 https://access.redhat.com/support/[Red Hat Customer Portal].
+endif::[]
+
+// TODO Need to add a section describing how Dedicated Admins can manage egress network policies.
```

Comment:
need to make a decision on whether you're calling it dedicated admin or customer admin

Reply:
link to https://docs.openshift.com/container-platform/3.11/admin_guide/managing_networking.html#admin-guide-controlling-egress-traffic